deploy/manifests/staging/service-template.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: docs-${TRAVIS_PULL_REQUEST}
+  namespace: pxdocs
+spec:
+  type: ClusterIP
```

Comment:
[nit] My test cluster is using NodePorts, though it prooooobably works just fine with ClusterIPs. I'm fine merging like this and we can change later if it becomes an issue.